erroranalysis/tests/test_surrogate_error_tree.py

```diff
@@ -57,11 +57,13 @@ def test_traverse_tree(self):
         diff = pred_y != y_test
         max_depth = 3
         num_leaves = 31
+        min_child_samples = 20
```

Comment:
In EconML this param is called `min_tree_leaf_samples` ([code](https://github.com/microsoft/responsible-ai-widgets/blob/main/responsibleai/responsibleai/_managers/causal_manager.py#L61)) and in LightGBM it's called `min_data_in_leaf` ([docs](https://lightgbm.readthedocs.io/en/latest/Parameters-Tuning.html)).
Not sure if we want to standardize with some convention?